specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -8318,6 +8952,253 @@
       ],
       "description": "The list dedicated host operation response."
     },
+    "CapacityReservationGroupInstanceView": {
+      "type": "object",
+      "properties": {
+        "capacityReservations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CapacityReservationInstanceViewWithName"
+          },
+          "readOnly": true,
+          "description": "List of instance view of the capacity reservations under the capacity reservation group."
+        }
+      }
+    },
+    "CapacityReservationGroupProperties": {
+      "type": "object",
+      "properties": {
+        "capacityReservations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResourceReadOnly"
+          },
+          "readOnly": true,
+          "description": "A list of references to all capacity reservations in the capacity reservation group."
```

Comment:
A list of all capacity reservations resource ids that belong to capacity reservation group.

Reply:
updated